lib/eclipse/Parser/Parser.cpp

```diff
@@ -484,10 +495,17 @@ bool tryParseKeyword( ParserState& parserState, const Parser& parser ) {
         if( parserState.rawKeyword == NULL ) {
             if( RawKeyword::isKeywordPrefix( line, keywordString ) ) {
                 parserState.rawKeyword = createRawKeyword( keywordString, parserState, parser );
+                parserState.lastSizeType = SLASH_TERMINATED;
```

Comment:
The init to `SLASH_TERMINATED` here should be removed.

Reply:
There may be a situation in which lastSizeType is set to OTHER_KEYWORD_IN_DECK. Then follows the correct number of records. THEN there are some correct keywords, followed by random gibberish. If the lastSizeType is not reset, this will be handled as a PARSE_EXTRA_RECORDS situation, which it is not. Example:
         EQLDIMS
           2  100  20  1  1  /
         EQUIL
           2469   382.4   1705.0  0.0    500    0.0     1     1      20 /
           2469   382.4   1705.0  0.0    500    0.0     1     1      20 /
         GRID
         randomgibberish

I will also include this as test.